examples/index.html

```diff
@@ -25,8 +26,12 @@ <h1>Leaflet BlurredLocation <small>| by <a href="">Public Lab</a></small></h1>
       <hr />
 
       <h3>Enter a location</h3>
-      
-      <p><button class="btn btn-primary" onclick="blurredLocation.geocodeWithBrowser();">Get my location</button></p>
```

Comment:
Ah, just take a look here at what a new contributor changed first -- this has become a button. Can you adapt your changes to a button? Thanks!!! This is going to be great.

Reply:
Not an issue just a moment!!